src/ansys/tools/common/launcher/_plugins.py

```diff
@@ -30,7 +30,8 @@
 
 from .interface import FALLBACK_LAUNCH_MODE_NAME, DataclassProtocol, LauncherProtocol
 
-LAUNCHER_ENTRY_POINT = "ansys.tools.local_product_launcher.launcher"
+LAUNCHER_ENTRY_POINT = "ansys.tools.common.launcher"
+DEPRECATED_LAUNCHER_ENTRY_POINT = "ansys.tools.local_product_launcher.launcher"
```

Comment:
Currently, the `ansys-launcher` command may refer to either the version from `ansys-tools-common`, or the one from `ansys-tools-local-product-launcher`, depending on which package was installed last.

That's unavoidable since we want to reuse the `ansys-launcher` name, but we can ease the transition by initially accepting both entry points. Then we can upgrade with the following steps (in relatively short succession):
- initial state: plugins use the old entry point
- deprecate `ansys-tools-local-product-launcher`:
  - it only re-exports the objects from `ansys-tools-common`
  - remove its script entrypoint
- plugins switch to the new entry point
- (some time later) remove the `DEPRECATED_LAUNCHER_ENTRY_POINT` handling here